data/en/1_0005.json

```diff
@@ -385,6 +380,54 @@
                         "title": "Weekly Pay",
                         "type": "Question"
                     },
+                    {
+                        "type": "Question",
+                        "title": "Weekly Pay",
+                        "id": "confirm-weekly-pay-gross-pay",
+                        "skip_conditions": [{
+                            "when": [{
+                                "id": "weekly-pay-gross-pay-answer",
+                                "condition": "greater than",
+                                "value": 0
+                            }]
+                        }],
+                        "questions": [{
+                            "type": "General",
+                            "answers": [{
+                                "type": "Radio",
+                                "id": "confirm-weekly-pay-gross-pay-answer",
+                                "q_code": "d50",
+                                "options": [{
+                                        "label": "Yes this is correct",
+                                        "value": "Yes"
+                                    },
+                                    {
+                                        "label": "No I need to change this",
+                                        "value": "No"
+                                    }
+                                ],
+                                "mandatory": true
+                            }],
+                            "id": "confirm-weekly-pay-gross-pay-question",
+                            "title": "The <em>total gross weekly pay</em> paid to employees in the last week of {{exercise.period_str}} was <em>£0</em>, is this correct?"
+                        }],
+                        "routing_rules": [{
+                                "goto": {
+                                    "when": [{
+                                        "value": "No",
+                                        "id": "confirm-weekly-pay-gross-pay-answer",
+                                        "condition": "equals"
+                                    }],
+                                    "block": "weekly-pay-gross-pay"
+                                }
+                            },
+                            {
+                                "goto": {
+                                    "group": "fortnightly-pay"
```

Comment:
@charlotte-hirst. Although the slide shows route to end is there an argument that we should actually route to the significant change question?

Reply:
I was thinking that earlier

Reply:
it should go to significant changes to explain why its zero